### ISSUE TITLE
Quokka fails to start on New File (Error: ENOENT)

### ISSUE THREAD

**Issue by beauallison** [wallabyjs/quokka#234]
### Issue description or question

After updating both VS Code + Quokka it's now failing to start (error below) on `New File` commands. Unsure whether this was the VS Code update or Quokka however I've replicated the issue by installing Quokka on a co-workers machine with the same versions.

### Versions

VS Code Version: `1.26.1`
Quokka Version: `1.0.137`

## Commands

`New Javascript File` - Error (as below)
`New TypeScript File` - Error (as below)

### Quokka.js Console Output

![image](https://user-images.githubusercontent.com/14225594/44259252-ec23b000-a208-11e8-966e-3e6ab6c5b5e2.png)

```
Quokka #1 (node: v8.11.3)​​​​
​​Failed to run preprocessors on quokka.js, Error: ENOENT: no such file or directory, open './quokka.js'​​
```

**Comment by beauallison** [comment]
I found a strange fix - something to do with the underlying node modules perhaps? The post install script?

1. Make a file `index.js` with similar code below (make sure it imports/requires)
```const fs = require('fs');```

2. Select the file and run `Start on Current File`

Quokka JS will now work as normal. If you have a file that's just `console.log('x')` this won't fix the issue.

**Comment by ArtemGovorov** [comment]
Thanks for the report. The issue is fixed and the fix is published in the latest v1.0.138.
